src/test/java/hudson/plugins/active_directory/ActiveDirectorySecurityRealmFipsAndForceLdapEnabledTest.java

```diff
@@ -0,0 +1,64 @@
+package hudson.plugins.active_directory;
+
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.recipes.LocalData;
+
+import hudson.util.FormValidation;
+
+import static org.junit.Assert.assertEquals;
+
+import static hudson.plugins.active_directory.ActiveDirectorySecurityRealm.LEGACY_FORCE_LDAPS_PROPERTY;
+
+/*
+Verifies the warning message when Jenkins operates in FIPS and Force LDAP property is enabled.
+ */
+public class ActiveDirectorySecurityRealmFipsAndForceLdapEnabledTest {
+
+    @Rule
+    public JenkinsRule jenkinsRule = new JenkinsRule();
+
+    @BeforeClass
+    public static void globalSetup() {
+        System.setProperty("jenkins.security.FIPS140.COMPLIANCE", "TRUE");
+        System.setProperty(LEGACY_FORCE_LDAPS_PROPERTY, "TRUE");
+    }
+
+    @LocalData
```

Comment:
No need for a LocalData, in this test you can use `ExtensionList` to find the Descriptor class using this class: https://github.com/jenkinsci/active-directory-plugin/blob/11d154fbe2bcdad7e8a05bdeefbff746c4d837f1/src/main/java/hudson/plugins/active_directory/ActiveDirectorySecurityRealm.java#L481